CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Adding Cache-Control max-age headers to all http responses set to 1h.
```

Comment:
@jfsiii @skh I plan to add now changelog entries for all PR's which affect the API in any way and probably most others too, except some refactoring.